Server/src/main/java/server/ServerFacade.java

```diff
@@ -44,6 +43,17 @@ public LoggedInResults loginUser(String username, String password) {
         loggedInResults.setAuthToken(authToken);
         loggedInResults.setSuccess(true);
 
+        String className = "iClient";
```

Comment:
I'm not sure this will work, since `iClient` isn't a class. Maybe it would have to be `ClientProxy`